### ISSUE TITLE
Thanks.

### ISSUE THREAD

**Issue by albert-flex** [Chris2011/netbeans-textmate-files#1]
Thanks to your works on NetBeans Language Servers,I also like NetBeans Very Much, and I'm appreciate your works very much!

**Comment by albert-flex** [comment]
However, I could not find the folder you just wrote on the readme.md that `client.properties` on `%userprofile%\AppData\Roaming\NetBeans\YourNetBeansVersion\config\Preferences\org\netbeans\modules\lsp`,
I could not find the lsp folder.
How should I do then? plz tell me when you are available.

**Comment by Chris2011** [comment]
Hi, thx for your ticket. Please give more information about your environment for example, which Apache NetBeans version do you use?. Best is to use version 12.0.

**Comment by albert-flex** [comment]
Netbeans 12.0 win10 jdk13.0.2,
and I can't find the ```lsp``` folder in my ```%userprofile%\AppData\Roaming\NetBeans\YourNetBeansVersion\config\Preferences\org\netbeans\modules```.

**Comment by Chris2011** [comment]
If you can't see the option, also the option "Editor" you need to open one Project first. That will initialize the ergonomics module in NetBeans and then you see the options. If you see the options then, you should find the file in your path. Otherwise let me know.

**Comment by albert-flex** [comment]
Yeah I see. I found that folder. let me have a try again, Thanks.

**Comment by Chris2011** [comment]
Cool, let me know whether it works or not :)

**Comment by albert-flex** [comment]
I am sure that I add the json data to that client.properties file, However , When I Open the NetBeans 12.0 to see the options there, nothing as the before. Also I changed the relative path to where I place netbeans-textmate-files, did not work.
How was going on?

**Comment by Chris2011** [comment]
Please make a screenshot from the options to let me know, how it looks like. And please share the clients.properties file with me :)

**Comment by Chris2011** [comment]
We also can do a remote session that I can help you :)

**Comment by albert-flex** [comment]
I'm Sorry,I forgot that. I had reinstall the netbeans, and I will Try Again at tonight.

**Comment by Chris2011** [comment]
Ok, just let me know :)

**Comment by albert-flex** [comment]
Oh,My God,I can not found that 'lsp' folder again. I did open three project.
![图片](https://user-images.githubusercontent.com/61842869/90576161-f578f600-e1ef-11ea-9eeb-975f85d72bd5.png)

**Comment by Chris2011** [comment]
Please let us talk in private, and let us check it out, this is not the right way. Join the NetBeans slack chan for that :). Please give me your mailaddress and I can invite you.

**Comment by albert-flex** [comment]
mailaddress: albert-flex@foxmail.com
